sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -457,20 +427,19 @@ client.listChatThreadMembers(chatThreadId: threadId) { result, _ in
 
 Use the `add` method to add members to a thread.
 
-- `AddChatThreadMembersRequest` is used to list the `ChatThreadMember`s to be added to the thread.
+- `AddChatParticipantsRequest` is used to list the `ChatParticipant`s to be added to the thread.
 - `id`, required, is the `CommunicationUser.identifier` you created before. Refer to [User and User Access Tokens](#user-and-user-access-tokens).
 - `displayName`, optional, is the display name for the thread member.
 - `shareHistoryTime`, optional, is the time from which the chat history is shared with the member.
-- `chatThreadId` is the unique ID of the thread.
 
 ```swift
-let threadMembers = AddChatThreadMembersRequest(
-    members: [ChatThreadMember(
+let threadParticipants = AddChatParticipantsRequest(
+    participants: [ChatParticipant(
         id: userId,
         displayName: "a new member"
     )]
 )
-client.add(chatThreadMembers: threadMembers, chatThreadId: threadId) { result, _ in
+chatThreadClient.add(chatParticipants: threadParticipants) { result, _ in
```

Comment:
Same here, perhaps just `add(participants:)`

Reply:
Also, in this case can we just make the parameter an array of `ChatParticipant`s? I don't see any value in wrapping the array with an `AddChatParticipantsRequest`.